### PR TITLE
Add privileged interface

### PR DESCRIPTION
**Pull request by semtexzv**
Use platform-go-middleware for parsing identity, Add a new api group accepting only associate auth, which comes from turnpike.

### CODE REVIEW

File: base/utils/identity.go [hunk 1]
```diff
@@ -3,50 +3,20 @@ package utils
...
 
-type Identity struct {
-	Entitlements map[string]Entitlement `json:"entitlements"`
-	Identity     IdentityDetail         `json:"identity"`
-}
-
-func ParseIdentity(identityString string) (*Identity, error) {
-	var ident Identity
 	decoded, err := base64.StdEncoding.DecodeString(identityString)
+	print(string(decoded))
```
Comment (Josca) [inline base/utils/identity.go:13]: print?

File: manager/middlewares/authentication.go [hunk 2]
```diff
@@ -43,17 +45,34 @@ func Authenticator() gin.HandlerFunc {
 		}
 		utils.Log("ident", identStr).Trace("Identity retrieved")
 
-		identity, err := utils.ParseIdentity(identStr)
+		ident, err := utils.ParseIdentity(identStr)
```
Comment (Josca) [inline manager/middlewares/authentication.go:48]: What's the purpose of this change?

**Review by Josca — CHANGES_REQUESTED** [review]
looks good, I think I found one forgotten print, thanks

File: manager/middlewares/authentication.go (hunk 2 above)
Comment (semtexzv) [reply manager/middlewares/authentication.go:48]: Conflict with package name

**Review by Josca — APPROVED** [review]

**Pull request merged**